cunumeric/coverage.py

```diff
@@ -129,10 +133,12 @@ def clone_module(
         omit_types=(ModuleType,),
     )
 
-    if runtime.report_coverage:
-        for attr, value in new_globals.items():
-            if isinstance(value, FunctionType):
+    for attr, value in new_globals.items():
+        if isinstance(value, FunctionType):
+            if runtime.report_coverage:
                 new_globals[attr] = implemented(value, module_name)
+            else:
+                value._cunumeric_implemented = True
```

Comment:
I would actually like to propose calling the `implemented` function unconditionally so that setting this attr can be handled in one place, but I will leave that for the future.

Reply:
yeah I also thought about doing that, and I feel we should do it now. let's make a follow-up PR for it.